digital-twin/Samples/device/JdkSample/pom.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>pom</packaging>
+    <groupId>com.microsoft.azure.sdk.iot.device.samples</groupId>
+    <artifactId>digital-twin-device-samples</artifactId>
+    <version>1.0.0</version>
+	<url>https://github.com/Azure-Samples/azure-iot-samples-java</url>
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://opensource.org/licenses/MIT</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <developers>
+        <developer>
+            <id>microsoft</id>
+            <name>Microsoft</name>
+        </developer>
+    </developers>
+    <scm>
+        <connection>scm:git:git@github.com:Azure-Samples/azure-iot-samples-java.git</connection>
+        <developerConnection>scm:git:git@github.com:Azure-Samples/azure-iot-samples-java.git</developerConnection>
+        <url>https://github.com/Azure-Samples/azure-iot-samples-java.git</url>
+    </scm>
+    <modules>
+        <module>environmental-sensor-sample</module>
+    </modules>
+    <properties>
+		<java-sdk-device-version>1.7</java-sdk-device-version>
+        <java-sdk-service-version>1.8</java-sdk-service-version>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <lombok-version>1.18.4</lombok-version>
+        <maven-compiler-plugin-version>3.8.1</maven-compiler-plugin-version>
+        <maven-source-plugin-version>3.1.0</maven-source-plugin-version>
+        <maven-shade-version>2.4</maven-shade-version>
+        <maven-jar-plugin-version>3.1.0</maven-jar-plugin-version>
+        <build-helper-version>3.0.0</build-helper-version>
+        <slf4j-version>1.7.25</slf4j-version>
+        <commons-io-version>2.6</commons-io-version>
+        <project-device-samples-version>1.0.0</project-device-samples-version>
+        <provisioning-device-client-version>LATEST</provisioning-device-client-version>
+		<digital-twin-device-client-version>LATEST</digital-twin-device-client-version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${lombok-version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure.sdk.iot</groupId>
+            <artifactId>digital-twin-device-client-preview</artifactId>
+            <version>${digital-twin-device-client-version}</version>
```

Comment:
why cant you specify "LATEST"here as well ?

Reply:
The value is `LATEST`